packages/website/src/api/character.ts

```diff
@@ -0,0 +1,11 @@
+import { request } from './request'
+import { CharacterDetail } from '../types/common'
+import { AxiosPromise } from 'axios'
+
+const api = {
+  getCharacterDetail: '/v0/characters/'
+}
+
+export const getCharacterDetail = (characterId:string|number):AxiosPromise<CharacterDetail> => {
```

Comment:
```suggestion
export const getCharacterDetail = (characterId:string):AxiosPromise<CharacterDetail> => {
```

最好确认只用一种类型，`string`或者`number`都可以，也可以单独抽出来一个类型

```typescript
type CharacterId = string;
```